src/context.cpp

```diff
@@ -645,12 +650,6 @@ namespace librealsense
         {
             start_device_watcher();
         }
-#ifdef BUILD_WITH_DDS
-        if( _dds_watcher && _dds_watcher->is_stopped() )
```

Comment:
Why would we want to stop the device watcher but not the DDS device watcher?

Reply:
This code section is not about stopping the device watcher, it's about turning it on.
When DDS is enabled we should start the watcher as soon as possible since the device creation is a `"long" process in comparation with USB device detection.
We want to start the sensors & profiles detection ASAP so when the user query for devices he will wait as less as possible.

If we find we need to change it in the future we can.
For now I think it's better.

Reply:
Hmm OK, makes sense. I suggest we put a comment to this effect (that we want the DDS device watcher always on, even when no callbacks) in context.cpp

Reply:
👍

Reply:
Done